src/apps/sorting-visualizer/components/tooltip/tooltip.module.scss

```diff
@@ -0,0 +1,37 @@
+// Tooltip.scss
+
+.tooltipContainer {
+    position: relative;
+    display: inline-block;
+    padding-top: 4px;
+  
+    &:hover .tooltip {
+      opacity: 1;
+      visibility: visible;
+    }
+  }
+  
+  .tooltip {
+    position: absolute;
+    background-color: #333;
+    color: #fff;
+    padding: 5px;
+    border-radius: 4px;
+    font-size: 12px;
```

Comment:
Font size can be in rem. When users increase the font, px will not increase accordingly.